readme.md

```diff
@@ -0,0 +1 @@
+This is my readme!
```

Comment:
Testing a code review. Change this to a heading with just "Angular Automation Exercise"